SConscript

```diff
@@ -35,7 +35,7 @@ def CheckPKG(context, name):
 
 # Generate a config.py file with build data.
 def create_config_py(env):
-    mypaint_brushdir = check_output(['pkg-config', '--variable=brushdir', 'mypaint-brushes'])
+    mypaint_brushdir = check_output(['pkg-config', '--variable=brushesdir', 'mypaint-brushes-2.0'])
```

Comment:
https://github.com/mypaint/mypaint-brushes/blob/7e1e89927863fbe499e9b49e9184a5faa6ef2498/pkgconfig.pc.in#L4